Source/Core/Core/IOS/USB/Bluetooth/BTEmu.h

```diff
@@ -57,13 +56,15 @@ class BluetoothEmu final : public BluetoothBase
 
   bool RemoteDisconnect(u16 _connectionHandle);
 
-  std::vector<WiimoteDevice> m_WiiMotes;
+  WiimoteDevice* AccessWiiMoteByIndex(std::size_t index);
   WiimoteDevice* AccessWiiMote(const bdaddr_t& _rAddr);
   WiimoteDevice* AccessWiiMote(u16 _ConnectionHandle);
```

Comment:
Can we make that one `private`? Basically same reason as bringing `AccessWiiMoteByIndex` to the party; hiding the implementation detail (in this case: the connection handle) from callers.

Reply:
Good call. Given neither this one, or the one taking `bdaddr_t` references are used externally, I've made those two private.